development/contracts/ForkableBridge.sol

```diff
@@ -320,4 +325,24 @@ contract ForkableBridge is IForkableBridge, ForkableUUPS, PolygonZkEVMBridge {
             metadata
         );
     }
+
+    ///////////////////////////////
+    /// View functions
+    ////////////////////////////////
+
+    function getHardAssetManger() external view returns (address) {
+        return _hardAssetManger;
+    }
+
+    function getLastUpdatedDepositCount() external view returns (uint32) {
```

Comment:
I could not just make lastUpdatedDepositCount an public variable, as this would declare an public function to get the variable, and the interface has to declare a external function.

It's kinda stupid. There should be a better solution, but this is seems to be the best one can do right now

Reply:
Yeah, I've run into this myself, I don't know a better way.